protocols/relay/src/priv_client/transport.rs

```diff
@@ -50,9 +50,9 @@ use thiserror::Error;
 ///    # use libp2p_core::{Multiaddr, multiaddr::{Protocol}, Transport, PeerId};
 ///    # use libp2p_core::transport::memory::MemoryTransport;
 ///    # use libp2p_core::transport::choice::OrTransport;
-///    # use libp2p_relay::v2::client;
+///    # use libp2p_relay::client;
```

Comment:
```suggestion
///    # use libp2p_relay as relay;
```

Let's be consistent with our other examples.

Reply:
yeah agreed! Updated :)